### ISSUE TITLE
Unable to deploy to k8s

### ISSUE THREAD

**Issue by shayrybak** [ansible/ansible-container#881]
##### ISSUE TYPE

 - Bug Report
##### container.yml 

```
version: "2"
settings:
  conductor:
    # The Conductor container does the heavy lifting, and provides a portable
    # Python runtime for building your target containers. It should be derived
    # from the same distribution as you're building your target containers with.
    base: centos:7
    roles_path: 
      - ../../roles

  project_name: test

services: 
  sumologic:
    from: centos:7
    expose:
      - 514
    entrypoint: "/usr/bin/dumb-init"
    command: /opt/SumoCollector/collector console -- -t
    roles:
      - dumb_init
      - sumologic
    user: root

registries: 
  k8s:
    url: https://127.0.0.1:8443
    namespace: test

```

##### OS / ENVIRONMENT

```
Darwin, Shays-MBP.localdomain, 17.4.0, Darwin Kernel Version 17.4.0: Sun Dec 17 09:19:54 PST 2017; root:xnu-4570.41.2~1/RELEASE_X86_64, x86_64
2.7.10 (default, Jul 15 2017, 17:16:57)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.31)] /Users/shay/virtualenvs/ansible-container/bin/python
{
  "ContainersPaused": 0,
  "Labels": [],
  "CgroupDriver": "cgroupfs",
  "ContainersRunning": 0,
  "ContainerdCommit": {
    "Expected": "89623f28b87a6004d4b785663257362d1658a729",
    "ID": "89623f28b87a6004d4b785663257362d1658a729"
  },
  "InitBinary": "docker-init",
  "NGoroutines": 41,
  "Swarm": {
    "ControlAvailable": false,
    "NodeID": "",
    "Error": "",
    "RemoteManagers": null,
    "LocalNodeState": "inactive",
    "NodeAddr": ""
  },
  "LoggingDriver": "json-file",
  "OSType": "linux",
  "HttpProxy": "docker.for.mac.http.internal:3128",
  "Runtimes": {
    "runc": {
      "path": "docker-runc"
    }
  },
  "DriverStatus": [
    [
      "Backing Filesystem",
      "extfs"
    ],
    [
      "Supports d_type",
      "true"
    ],
    [
      "Native Overlay Diff",
      "true"
    ]
  ],
  "OperatingSystem": "Docker for Mac",
  "Containers": 63,
  "HttpsProxy": "docker.for.mac.http.internal:3129",
  "BridgeNfIp6tables": true,
  "MemTotal": 12575907840,
  "SecurityOptions": [
    "name=seccomp,profile=default"
  ],
  "Driver": "overlay2",
  "IndexServerAddress": "https://index.docker.io/v1/",
  "ClusterStore": "",
  "InitCommit": {
    "Expected": "949e6fa",
    "ID": "949e6fa"
  },
  "GenericResources": null,
  "Isolation": "",
  "SystemStatus": null,
  "OomKillDisable": true,
  "ClusterAdvertise": "",
  "SystemTime": "2018-02-07T09:01:10.079677883Z",
  "Name": "linuxkit-025000000001",
  "CPUSet": true,
  "RegistryConfig": {
    "AllowNondistributableArtifactsCIDRs": [],
    "Mirrors": [],
    "IndexConfigs": {
      "docker.io": {
        "Official": true,
        "Name": "docker.io",
        "Secure": true,
        "Mirrors": []
      }
    },
    "AllowNondistributableArtifactsHostnames": [],
    "InsecureRegistryCIDRs": [
      "127.0.0.0/8"
    ]
  },
  "DefaultRuntime": "runc",
  "ContainersStopped": 63,
  "NCPU": 4,
  "NFd": 22,
  "Architecture": "x86_64",
  "KernelMemory": true,
  "CpuCfsQuota": true,
  "Debug": true,
  "ID": "VDHN:PUWI:UIXZ:HPIN:RAQW:R66F:FMAN:2YIE:2K7A:BBMH:DRUZ:RTLV",
  "IPv4Forwarding": true,
  "KernelVersion": "4.9.60-linuxkit-aufs",
  "BridgeNfIptables": true,
  "NoProxy": "",
  "LiveRestoreEnabled": false,
  "ServerVersion": "17.12.0-ce",
  "CpuCfsPeriod": true,
  "ExperimentalBuild": true,
  "MemoryLimit": true,
  "SwapLimit": true,
  "Plugins": {
    "Volume": [
      "local"
    ],
    "Network": [
      "bridge",
      "host",
      "ipvlan",
      "macvlan",
      "null",
      "overlay"
    ],
    "Authorization": null,
    "Log": [
      "awslogs",
      "fluentd",
      "gcplogs",
      "gelf",
      "journald",
      "json-file",
      "logentries",
      "splunk",
      "syslog"
    ]
  },
  "Images": 76,
  "DockerRootDir": "/var/lib/docker",
  "NEventsListener": 2,
  "CPUShares": true,
  "RuncCommit": {
    "Expected": "b2567b37d7b75eb4cf325b77297b140ea686ce8f",
    "ID": "b2567b37d7b75eb4cf325b77297b140ea686ce8f"
  }
}
{
  "KernelVersion": "4.9.60-linuxkit-aufs",
  "Components": [
    {
      "Version": "17.12.0-ce",
      "Name": "Engine",
      "Details": {
        "KernelVersion": "4.9.60-linuxkit-aufs",
        "Os": "linux",
        "BuildTime": "2017-12-27T20:12:29.000000000+00:00",
        "ApiVersion": "1.35",
        "MinAPIVersion": "1.12",
        "GitCommit": "c97c6d6",
        "Arch": "amd64",
        "Experimental": "true",
        "GoVersion": "go1.9.2"
      }
    }
  ],
  "Arch": "amd64",
  "BuildTime": "2017-12-27T20:12:29.000000000+00:00",
  "ApiVersion": "1.35",
  "Platform": {
    "Name": ""
  },
  "Version": "17.12.0-ce",
  "MinAPIVersion": "1.12",
  "GitCommit": "c97c6d6",
  "Os": "linux",
  "Experimental": true,
  "GoVersion": "go1.9.2"
}
```
##### SUMMARY

after creating deployment playbook.

Running ansible-playbook ansible-deplyoment/test.yaml --tags start
I get the following error
##### STEPS TO REPRODUCE

```

```

##### ACTUAL RESULTS

```
 [WARNING]: Unable to parse /etc/ansible/hosts as an inventory source

 [WARNING]: No inventory was parsed, only implicit localhost is available

 [WARNING]: provided hosts list is empty, only localhost is available. Note
that the implicit localhost does not match 'all'


PLAY [Manage the lifecycle of test on K8s] *************************************

TASK [Create namespace test] ***************************************************
ok: [localhost]

TASK [Create service] **********************************************************
ok: [localhost]

TASK [Create deployment, and scale replicas up] ********************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: Exception: Error initializing AnsibleModuleHelper: {"message": "Error: Apps_v1beta1Deployment was not found in client.models. Did you specify the correct Kind and API Version?"}
fatal: [localhost]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/var/folders/69/63n5td7x39nc0vt9y2m28fbw0000gn/T/ansible_CU89cM/ansible_module_k8s_apps_v1beta1_deployment.py\", line 4751, in <module>\n    main()\n  File \"/var/folders/69/63n5td7x39nc0vt9y2m28fbw0000gn/T/ansible_CU89cM/ansible_module_k8s_apps_v1beta1_deployment.py\", line 4742, in main\n    raise Exception(exc.message)\nException: Error initializing AnsibleModuleHelper: {\"message\": \"Error: Apps_v1beta1Deployment was not found in client.models. Did you specify the correct Kind and API Version?\"}\n", "module_stdout": "", "msg": "MODULE FAILURE", "rc": 0}
	to retry, use: --limit @/Users/shay/projects/ansible-container/projects/deploy-test/ansible-deployment/test.retry

PLAY RECAP *********************************************************************
localhost                  : ok=2    changed=0    unreachable=0    failed=1

```

**Comment by re-mscho** [comment]
Have you installed the [OpenShift Rest Client](https://github.com/openshift/openshift-restclient-python) which is required by https://github.com/ansible/ansible-kubernetes-modules?

**Comment by shayrybak** [comment]
@re-mscho Thank you.

It was installed but on version 0.0.1.

After upgrading to the latest version everything worked as expected.

Maybe this should be added as a dependency for ansible-container with the minimum version?

**Comment by j00bar** [comment]
This was fixed in #877.
